cmd/calyptia/create_core_instance_gcp.go

```diff
@@ -154,6 +160,8 @@ func newCmdCreateCoreInstanceOnGCP(config *config, client gcp.Client) *cobra.Com
 	fs.StringVar(&githubToken, "github-token", os.Getenv("GITHUB_TOKEN"), "GitHub token for test purposes")
 	fs.BoolVar(&useTestImages, "use-test-images", envBool("CALYPTIA_USE_TEST_IMAGES"), "Use GCP test images instead of released channel (only for testing/development).")
 	fs.StringVar(&credentials, "credentials", os.Getenv("GOOGLE_APPLICATION_CREDENTIALS"), "Path to GCP credentials file. (default is $GOOGLE_APPLICATION_CREDENTIALS)")
+	fs.DurationVar(&rateLimit, "rate-limit", 20, "Rate limit for operations")
```

Comment:
Need to indicate the units here, is this rate per second, etc.?

Reply:
Duration flag accepts with unit.
2s, 5m, 1h29m4s, etc

Reply:
I think the default values should say
```
20 * time.Second
```
Tho, otherwise they are nanoseconds.